apps/music/test/marionette/metadata_test.js

```diff
@@ -5,6 +5,11 @@ var assert = require('assert');
 var Music = require('./lib/music.js');
 var ListviewHelper = require('./lib/listviewhelper.js');
 
+function assertStartsWith(start, actual) {
+  assert.equal(actual.indexOf(start), 0,
```

Comment:
String.prototype.startsWith() is a thing now:

assert.ok(actual.startsWith(start), 'message')